ui/component/publish/livestream/publishLivestream/view.jsx

```diff
@@ -0,0 +1,261 @@
+// @flow
```

Comment:
- [ ] Missing scheduling section when editing a livestream placeholder.

When editing a livestream placeholder, I see the "Edit/Update" tab now.
However, it is missing the scheduling section "When do you want to go live?".

Reply:
- [ ] "Edit livestream" shouldn't show "Release Date"?

Current production does not show this.  I tried editing it to 3 days ago, and the claim became "Live 3 days ago" despite I just went live today. (I'm not really sure what's the intended behavior -- just comparing against production)

Reply:
It seems like the tab is locked to edit mode when "Update or publish replay" is pressed.  
The only way to publish now is to hit "Clear", but after that, the user would need to return to the claim and hit "Update or publish replay" to edit because the tab is gone.

If it's intentional, I think I can understand why -- makes sense for the most part.  Just highlighting it here in case anyone else feels differently. @tzarebczan?

Reply:
Okay, so editing a scheduled Livestream now lets you update the release date and editing a replay switches over to the upload form.

Reply:
- [ ] When I try to publish a replay from scratch, "When do you want to go live?" appears.  Should the regular release date component appear instead here?